cmd/remove.go

```diff
@@ -0,0 +1,70 @@
+package cmd
+
+import (
+	"fmt"
+	"path/filepath"
+
+	"github.com/spf13/cobra"
+	"github.com/wtsi-hgi/go-softpack-builder/remove"
+	"github.com/wtsi-hgi/go-softpack-builder/s3"
+)
+
+var removeCmd = &cobra.Command{
+	Use:   "remove",
+	Short: "Remove an environment",
+	Long: `Remove an environment.
+
+Remove an existing environment; it's entry in the Core artefacts repo, the
+module files and the singularity image and symlinks.`,
+	Run: func(cmd *cobra.Command, args []string) {
+		if len(args) < 1 {
+			die("require a environment path")
+		}
+
+		const maxArgs = 2
+
+		if len(args) < maxArgs {
+			die("require a environment version")
+		}
+
+		if len(args) != maxArgs {
+			die("unexpected arguments")
+		}
+
+		conf := getConfig()
+
+		s, err := s3.New(conf.S3.BuildBase)
+		if err != nil {
+			die(err.Error())
+		}
+
+		envPath := filepath.Clean(string(filepath.Separator) + args[0])[1:]
+
+		if envPath != args[0] {
+			die("invalid environment path")
+		}
+
+		fmt.Printf(
```

Comment:
Do we have a cliPrint method in root? If not, copy it from one of our other repos. It's good to avoid fmt.Printf in "real" code, because it's often used for debugging and you want to do a quick search to ensure you removed all your debug.